### ISSUE TITLE
pre-deploy task is not working as build does not have correct labels 

### ISSUE THREAD

**Issue by mtamboli** [kabanero-io/kabanero-pipelines#384]
pre-deploy enforce policy is not getting enforced. This is due to old version of appsody not applying the correct labels.
```
[WARNING] The image label 'dev.appsody.stack.digest' has not been set for image: image-registry.openshift-image-registry.svc:5000/kabanero/appsodysb2:ed44d7cc9c66ea836662d95144af53a724f27b21, unable to enforce stackPolicy
```

**Comment by aadeshpa** [comment]
PR with this fix is : https://github.com/kabanero-io/kabanero-pipelines/pull/385

**Comment by aadeshpa** [comment]
@mtamboli : The fix for this issue is in 0.9.0-rc.6 of pipelines, can you please verify once and mark it as closed.

**Comment by mtamboli** [comment]
verified.
